apps/docs/pages/guides/realtime/quickstarts/postgres-changes.mdx

```diff
@@ -328,7 +471,36 @@ const channel = supabase
   .subscribe()
 ```
 
-## Custom Tokens
+## Full `old` record
+
+By default, only `new` record changes are sent but if you want to receive the `old` record (previous values) whenever you `UPDATE` or `DELETE` a record,
+you can set the `replica identity` of your table to `full`:
+
+```sql
+alter table
+  messages replica identity full;
+```
+
+<Admonition type="caution">
+
+RLS policies are not applied to `DELETE` statements. When RLS is enabled and `replica identity` is set to `full` on a table, the `old` record contains only the primary key(s).
+
+</Admonition>
+
+## Private schemas
+
+Postgres Changes works out of the box for tables in the `public` schema. You can listen to tables in your private schemas by granting table `SELECT` permissions to the database role found in your access token. You can run a query similar to the following:
+
+```sql
+grant select on "non_private_schema"."some_table" to authenticated;
+```
+
+<Admonition type="caution">
+  We strongly encourage you to enable RLS and create policies for tables in private schemas.
+  Otherwise, any role you grant access to will have unfettered read access to the table.
+</Admonition>
```

Comment:
```suggestion
<Admonition type="caution">

We strongly encourage you to enable RLS and create policies for tables in private schemas. Otherwise, any role you grant access to will have unfettered read access to the table.

</Admonition>
```

Here as well.